src/commonMain/kotlin/baaahs/sim/FakeNetwork.kt

```diff
@@ -111,6 +120,69 @@ class FakeNetwork(
             }
         }
 
+        private val mdns = FakeMdns()
+
+        inner class FakeMdns() : Network.Mdns {
+
+            override fun register(
+                hostname: String,
+                type: String,
+                proto: String,
+                port: Int,
+                domain: String,
+                params: MutableMap<String, String>
+            ): Network.MdnsRegisteredService? {
+                val fullname = "$hostname.$type.$proto.${domain.normalizeMdnsDomain()}"
+                val inst = FakeRegisteredService(hostname, type, proto, port, domain.normalizeMdnsDomain(), params)
+                services[fullname] = inst
+                (inst as? FakeRegisteredService)?.announceResolved()
```

Comment:
`inst` is always a `FakeRegisteredService` I think.

Reply:
This one, yeah, can go away.